dealers_choice.js

```diff
@@ -11,26 +11,313 @@ app.use(staticMiddleWare);
+          </ul>
+          <ul>
+            <li class="link" style="float:right"><a class="active" href="https://github.com/stephen-wong-tx/dealers_choice_js" target="_blank">About</a></li>
+            <li id="icon"><a href="/"><img src="/fourteeners-home-alt.png" alt="The number 14 in front of a mountain range drawing"></a></li>
+          </ul>
       </div>
+      <div id="main-content">
       <h1>The Fourtneeners</h1>
+      <p class="description"> <strong>Fourteener</strong> fȯr-ˈtēn-ər<br /> A mountain peek with an elevation of at least 14,000 ft (4267 m).</p>
+        <div id="card-container">
+          <div class="regular card">
+            <h2><i class="fas fa-mountain"></i>
+            <br />Mountain Ranges</h2>
+            Choose by location
+              <div class="checkboxContainer">
+                <div class="check-box">
+                  <input type="checkbox" id="1" name="Elk Mountains" value="Elk Mountains">
+                  <label for="Elk Mountains">Elk Mountains</label>
+                </div>
+                <div class="check-box">
+                  <input type="checkbox" id="6" name="Sawatch Range" value="Sawatch Range">
+                  <label for="Sawatch Range">Sawatch Range</label>
+                </div>
+                <div class="check-box">
+                  <input type="checkbox" id="5" name="Sangre De Cristo Range" value="Sangre De Cristo Range">
+                  <label for="Sangre De Cristo Range">Sangre De Cristo Range</label>
+                </div>
+                <div class="check-box">
+                  <input type="checkbox" id="3" name="Mosquito Range" value="Mosquito Range">            
+                  <label for="Mosquito Range">Mosquito Range</label>
+                </div>
+                <div class="check-box">
+                  <input type="checkbox" id="4" name="San Juan Mountains" value="San Juan Mountains">
+                  <label for="San Juan Mountains">San Juan Mountains</label>
+                </div>
+                <div class="check-box">
+                  <input type="checkbox" id="2" name="Front Range" value="Front Range">                
+                  <label for="Front Range">Front Range</label>
+                </div>
+              </div>
+          </div>
+          <div class="regular card">
+            <h2><i class="fas fa-tachometer-alt"></i> 
+            <br />
+            Skill Levels</h2>
+            Find climbs from beginner to expert
+            <div class="slidecontainer">
+              <input type="range" min="1" max="4" value="1" class="slider" id="myRange">
+              <ul id="slider-ul">
+                <li id="easy" value="1">Easy</li>
+                <li id="intermediate" value="2">Intermediate</li>
+                <li id="hard" value="3">Hard</li>
+                <li id="extreme" value="4">Extreme</li>
+              </ul>
```

Comment:
i think you have too many selectors- try reusing some of the selectors so you can limit the amount of repetition in your code